### PR TITLE
Fixes #464

### LINKED ISSUE #464: Config option can't find config file
| Q               | A
| --------------- | ---
| Version         | `13.1`
| Bug            | yes

**Steps to reproduce:**
```sh
$ cat grumphp.yml.dist
# output
parameters:
  bin_dir: "./vendor/bin"
  git_dir: "."
...

$ grumphp -c grumphp.yml.dist run
```

**Result:**
```
PHP Fatal error:  Uncaught InvalidArgumentException: The file "grumphp.yml.dist" does not exist (in: grumphp/src/Configuration/../../resources/config). in grumphp/vendor/symfony/config/FileLocator.php:68
```

### PR DESCRIPTION
**Pull request by janvernieuwe**
| Q             | A
| ------------- | ---
| Branch        | master
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Documented?   |no
| Fixed tickets | #464

Make the -c param work as intended.

### CODE REVIEW

File: src/Console/Application.php [hunk 1]
```diff
@@ -218,4 +223,19 @@ protected function initializeComposerHelper()
...
         return $this->composerHelper = new Helper\ComposerHelper($configuration, $rootPackage);
     }
+
+    /**
+     * Prefixes the cwd to the path given by the user
+     *
+     * @param string $configPath
+     * @return string
+     */
+    private function updateUserConfigPath(string $configPath): string
```
Comment (Landerstraeten) [inline src/Console/Application.php]: I don't think this is compatible with PHP7 😁

File: src/Console/Application.php [hunk 2]
```diff
@@ -119,17 +119,21 @@ protected function getDefaultHelperSet()
...
-            $container->get('grumphp.util.filesystem'),
-            $container->get('locator.external_command'),
-            $this->getDefaultConfigPath()
-        ));
-        $helperSet->set(new Helper\TaskRunnerHelper(
-            $container->get('config'),
-            $container->get('task_runner'),
-            $container->get('event_dispatcher')
-        ));
+        $helperSet->set(
```
Comment (Landerstraeten) [inline src/Console/Application.php]: I would keep this code block as it is. But thats me 🙂

File: src/Console/Application.php [hunk 3]
```diff
@@ -218,4 +223,19 @@ protected function initializeComposerHelper()
 
         return $this->composerHelper = new Helper\ComposerHelper($configuration, $rootPackage);
     }
+
+    /**
+     * Prefixes the cwd to the path given by the user
+     *
+     * @param string $configPath
+     * @return string
```
Comment (Landerstraeten) [inline src/Console/Application.php]: Not sure if an enter is required above '@return'. If not, just ignore this

Comment (SerkanYildiz) [reply src/Console/Application.php]: It is needed.

**Comment by janvernieuwe** [comment]
I undid the auto-formatting changes

**Pull request merged**